requirements.txt

```diff
@@ -3,4 +3,4 @@ setuptools>=19.2
 # Do *not* use 0.3 which is not backwards compatible to common clang versions
 libclang-py3==0.2
 appdirs==1.4.*
-coala_decorators>=0.2.1,==0.2.*
+coala_utils>=0.4.7
```

Comment:
please use `~=`, we should switch back to that anyway. The `>=...,==` combination expresses the same btw. but it doesn't have the benefit I hoped so let's use `~=`